### PR TITLE
Added files

### PR DESCRIPTION
**Pull request by chernetskyi8704**
### DOM

[Demo](https://chernetskyi8704.github.io/Interactive-Side-menu/) |
[Code base](https://github.com/chernetskyi8704/Interactive-Side-menu)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/chernetskyi8704/Interactive Side-menu/script.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+];
+
+iconBurger.addEventListener("click", (e) => {
+  document.body.classList.toggle("_lock");
+  iconBurger.classList.toggle("_active");
+  menuBody.classList.toggle("_active");
+});
+
+menuItem.forEach((element, index) => {
+  element.addEventListener("click", function () {
```
Comment (madmaxWMFU) [inline submissions/chernetskyi8704/Interactive Side-menu/script.js:53]: Read about [event delegation](https://javascript.info/event-delegation)

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work. See some comments below.
```javascript

 const renderContent = (pasta) => {
     const {name, image, .....} = pasta;
    const template = `${name} .....`;
     //insert to mainContentWrap 
 }

  const getContent = ({ target }) => {
      if( .... ) {
           const selectedPasta = contentData.find(pasta => pasta.name === target.innerText);
           renderContent(selectedPasta);
      }
  }
  menu.addEventListener('click', getContent)

```

File: submissions/chernetskyi8704/Interactive Side-menu/script.js (hunk 1 above)
Comment (chernetskyi8704) [reply submissions/chernetskyi8704/Interactive Side-menu/script.js]: Thank you for your comment. I think I made it.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Do a little change in the code. 
Separate event listener and function. 
Create separate functions:
- renderData()
- getSelectData()
- cleanContent()
- toglgleActiveMenuItem()

**Comment by chernetskyi8704** [comment]
Thanks for the comment. I was trying to figure out how to implement this functionality and got this, I'm not quite sure if it's right, but I couldn't think of any other way.

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Pull request merged**